src/figuresClasses.ts

```diff
@@ -1,19 +1,75 @@
+type Shape = 'triangle' | 'circle' | 'rectangle';
```

Comment:
```suggestion
type Shape = 'triangle' | 'circle' | 'rectangle';
```

better use enum here

Reply:
ok, done